### ISSUE TITLE
[BUG]: Regression with Anti-Blur in some situations (and when upscaling) (MGS2)

### ISSUE THREAD

**Issue by jonasos3** [PCSX2/pcsx2#8927]
### Describe the Bug

There seems to be a sort of regression, an issue similar to #6411 is back. When viewing transparent effects (codec with Otacons glasses, water surface, throwing a punch (the blur)) the image skips a line up or down, resulting in a jarring visual effect. If you look at any UI-element and press the punch button, you'll see the skip I'm talking about, same as pictured in #6411. Sort of like an interlace/line-skip.

The issue persists as long as any such effect is visible, and returns to normal after. Throw a O O O combo to see it change back and forth rapidly, here it is very noticable.

### Reproduction Steps

Use newest nightly (1.7.4549).
Load Metal Gear Solid 2 (I've used the Substance NTSC version).
Start new game (tanker first). Skip cutscenes, get to the tanker/playable section.
Punch, see how the image skips up/down.

### Expected Behavior

On older versions (1.7.4035 for example) the issue is there as well, BUT turning anti-blur off sorts it out. This is also the case for the newer builds, HOWEVER, not when upscaling.

When upscaled and anti-blur off, new builds still skip up/down.
When upscaled and anti-blur off, old builds DO NOT skip.

Not skipping is obviously preferable.

### PCSX2 Revision

1.7.4549

### Operating System

Windows 11

### CPU

i9-13900

### GPU

RTX 4080

### GS Settings

Everything at default, only changing Anti-Blur, Upscale/render-resolution.

**Comment by refractionpcsx2** [comment]
Yeah, I kinda re-broke it when I rewrote the PCRTC, I guess I can look at it again, I just don't like adding all these hacks to get around problems :(

**Comment by jonasos3** [comment]
Thanks for taking a look if you do find the time, I appreciate it very much either way! :)

**Comment by jonasos3** [comment]
Just a thought, sort of bypassing the issue for now; should Anti-Blur OFF be added to the GameDB/gamefixes for MGS2? This way the problem will only appear when upscaling (at least until it is fixed), and the annoying bobbing/skipping will be gone. This way new players on default settings won't experience the issue :)

Even before it was "re-broke" as you said, the bobbing would be there with Anti-Blur ON.

**Comment by jonasos3** [comment]
^...

"Even before it was "re-broke" as you said, the bobbing would be there with Anti-Blur ON." i thought, but I was informed that this shouldn't be the case. That there were builds where Anti-Blur could be ON, and there would be no bobbing.

Just to get this information posted somewhere, I've now tested builds;
v1.7.2980
v1.7.3603
v1.7.3816
v1.7.3963
v1.7.4011
v1.7.4035
v1.7.4039
v1.7.4558
on two different PC's (one Windows 11, one Windows 10). Freshly downloaded builds, extracted to their own folders, default settings, tried both NTSC and PAL versions of the game MGS2, and I cannot find a build that does not display this bobbing while Anti-Blur is ON, even after the previous fix (which was now re-broken). I was under the impression that the pervious fix included allowing users to disable Anti-Blur, making the bobbing a non-issue (although also making the game blurrier), not that it would allow for Anti-Blur to be ON while not getting the bobbing.

If anyone has any of the builds that should allow for Anti-Blur ON + no bobbing I would love to try them out, I can't get it to work at least on my end... Thanks!
